requirements-ci-current.txt

```diff
@@ -402,22 +403,22 @@ pygments==2.15.1
     #   sphinx
 pymess==1.0.0
     # via pymor (pyproject.toml)
-pyopengl==3.1.6
+pyopengl==3.1.7
```

Comment:
Not sure why, but CI failures suggest that there is something wrong with the newest pyopengl version. Can you try pinning it to an older version? If that resolves the issue, I'd leave the pin and open an issue to check the situation after the release.

Reply:
I restricted the version, let's see.
I tried it locally with `pyopengl==3.1.7`, and `xvfb-run pytest` fails, but `pytest` by itself seems to work.